fabtests/benchmarks/benchmark_shared.c

```diff
@@ -294,26 +313,32 @@ int bandwidth_rma(enum ft_rma_opcodes rma_op, struct fi_rma_iov *remote)
 	inject_size = inject_size_set ?
 			hints->tx_attr->inject_size: fi->tx_attr->inject_size;
 
-	if (opts.options & FT_OPT_ENABLE_HMEM)
+	if (ft_check_opts(FT_OPT_ENABLE_HMEM) || ft_check_opts(FT_OPT_VERIFY_DATA))
```

Comment:
Probably need to explain why inject_size was set to 0 when verify data is requested.

Reply:
as comment

Reply:
done.